README.md

```diff
@@ -460,6 +459,13 @@ List of variable names you may change within the `placeholders`property:
 | date     | `string` | Date input placeholder.       | Enter date...       |
 | file     | `string` | File input placeholder.       | Select file...      |
 
+List of variable names you may change within the `pagination`property:
```

Comment:
missing a space between `pagination` and property.

Reply:
Fixed